### PR TITLE
[Merged by Bors] - chore: fix or update three porting notes referring to old nightlies

### PR DESCRIPTION
**Pull request by grunweg**
These nightlies have since been released.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Order/Monoid/OrderDual.lean [hunk 1]
```diff
@@ -88,10 +88,8 @@ instance orderedCommMonoid [OrderedCommMonoid α] : OrderedCommMonoid αᵒᵈ :
 @[to_additive OrderDual.OrderedCancelAddCommMonoid.to_contravariantClass]
 instance OrderedCancelCommMonoid.to_contravariantClass [OrderedCancelCommMonoid α] :
     ContravariantClass αᵒᵈ αᵒᵈ HMul.hMul LE.le where
-    -- Porting note: We need to specify the implicit arguments here because of
-    -- https://github.com/leanprover/lean4/issues/1892
-    -- We should be able to remove this after nightly-2022-11-30 arrives.
-    elim a b c := @OrderedCancelCommMonoid.le_of_mul_le_mul_left α _ a c b
+    -- Porting note: need to specify `α`; not sure why
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Order/Monoid/OrderDual.lean:91]: This seems unsurprising since we're proving a statement about `αᵒᵈ`. I'd remove the note entirely

**Review by Ruben-VandeVelde — APPROVED** [review]

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

File: Mathlib/Algebra/Order/Monoid/OrderDual.lean [hunk 2]
```diff
@@ -88,10 +88,7 @@ instance orderedCommMonoid [OrderedCommMonoid α] : OrderedCommMonoid αᵒᵈ :
 @[to_additive OrderDual.OrderedCancelAddCommMonoid.to_contravariantClass]
 instance OrderedCancelCommMonoid.to_contravariantClass [OrderedCancelCommMonoid α] :
     ContravariantClass αᵒᵈ αᵒᵈ HMul.hMul LE.le where
-    -- Porting note: We need to specify the implicit arguments here because of
```
Comment (mattrobball) [inline Mathlib/Algebra/Order/Monoid/OrderDual.lean:91]: Why was this deleted? We still need to make the implicit explicit?

Comment (grunweg) [reply Mathlib/Algebra/Order/Monoid/OrderDual.lean:91]: @Ruben-VandeVelde suggested this - please discuss among each other; I have no opinion here :-)

Comment (Ruben-VandeVelde) [reply Mathlib/Algebra/Order/Monoid/OrderDual.lean:91]: I don't think there's anything actionable about this note - even if we could drop the argument, I would probably keep it for robustness.

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge
